pghoard/transfer.py

```diff
@@ -51,6 +51,7 @@ def defaults():
                 return {
                     "basebackup": EMPTY.copy(),
                     "basebackup_chunk": EMPTY.copy(),
+                    "basebackup_delta": EMPTY.copy(),
```

Comment:
Is it a enum?

Reply:
My goal is not to refactor everything :)